frontend/src/molecules/lists/drag-and-drop-list-box.tsx

```diff
@@ -98,11 +99,19 @@ const DragAndDropListBox = <T extends any>(
         const cssIsDragging = (snapshot: DraggableStateSnapshot) =>
             snapshot.isDragging ? "-dragging" : "";
 
+        const itemIdToDraggableId = (itemId: T): DraggableId => {
+            const itemIdAsObject: object = itemId as object;
+            if (itemIdAsObject?.toString() != null) {
```

Comment:
Remove the parenthesis so as not to invoke `toString`. We want to check that the function itself exists, not that its return exists.

My bad on this one, I made the change and never updated the snippet.